src/node/ext/server.cc

```diff
@@ -147,6 +175,13 @@ bool Server::HasInstance(Local<Value> val) {
   return Nan::New(fun_tpl)->HasInstance(val);
 }
 
+void Server::DestroyWrappedServer() {
```

Comment:
related to below comment, the shutdown is wrapped in the null check in the callback but isn't in `TryShutdown`?

mostly wondering if `TryShutdown` can be more similar with `ForceShutdown`